mne_bids_pipeline/_io.py

```diff
@@ -16,9 +13,3 @@ def _write_json(fname: PathLike, data: dict) -> None:
 def _read_json(fname: PathLike) -> dict:
     with open(fname, encoding="utf-8") as f:
         return json_tricks.load(f)
-
-
-def _empty_room_match_path(run_path: BIDSPath, cfg: SimpleNamespace) -> BIDSPath:
```

Comment:
Moved this to `_import_data.py` which is a better place for it. `_io.py` now just has to do with generic file I/O (currently just `json`) rather than our chose (BIDS)Path structures.